assets/dev/scss/frontend/conditionals/with-breakpoints/lightbox.scss

```diff
@@ -242,9 +242,19 @@
 		}
 
 		@media (max-width: $screen-tablet-max) {
-			&:active {
-				-webkit-tap-highlight-color: transparent; /* fallback for some Androids */
-				-webkit-tap-highlight-color: rgba( 0, 0, 0, 0 );
+			&-prev, &-next {
```

Comment:
```suggestion
			&-prev,
			&-next {
```

Reply:
Usually we like to break it into multiple lines. I am seeing that hasn't been done consistently, but I would be nice if we stick to convention.